### PR TITLE
[Components with more than 1 service] LPS-195490 Refactor EditServerMVCActionCommand component

### PR DESCRIPTION
**Pull request by RenanUmV**

### CODE REVIEW

File: modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/portlet/action/EditServerMVCActionCommand.java [hunk 1]
```diff
@@ -894,25 +869,24 @@ private void _verifyMembershipPolicies() throws Exception {
...
 	private static final Function<InvocationHandler, Layout>
 		_proxyProviderFunction = ProxyUtil.getProxyProviderFunction(
 			Layout.class, ModelWrapper.class);
 	private static final MethodKey _resetLogLevelsMethodKey = new MethodKey(
 		EditServerMVCActionCommand.class, "_resetLogLevels", Map.class,
 		Map.class);
 	private static final MethodKey _updateLogLevelsMethodKey = new MethodKey(
-		EditServerMVCActionCommand.class, "_updateLogLevels", Map.class,
-		String.class);
+		EditServerMVCActionCommand.class, "_updateLogLevels", Map.class);
```
Comment (rafaprax) [inline modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/portlet/action/EditServerMVCActionCommand.java:885]: @RenanUmV Why did you change the method reference?

Comment (rafaprax) [reply modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/portlet/action/EditServerMVCActionCommand.java:885]: Can you please point to the pattern you are following here?

Comment (RenanUmV) [reply modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/portlet/action/EditServerMVCActionCommand.java:885]: I changed the method reference because the method that was used before was deleted, in the method I'm using now it only receives a Map as a parameter. I'm following the pattern that Dante applied in this [PR](https://github.com/liferay-core-infra/liferay-portal/pull/4718/commits/4fded4a90a29f6e7b7aab446185156bab565b7b5)

Comment (rafaprax) [reply modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/portlet/action/EditServerMVCActionCommand.java:885]: got it

Comment (rafaprax) [reply modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/portlet/action/EditServerMVCActionCommand.java:885]: send it to core-infra repo ant explain that there as well

**Review by rafaprax — APPROVED** [review]
